clients/fides-js/src/components/Overlay.tsx

```diff
@@ -100,7 +102,16 @@ const Overlay: FunctionComponent<Props> = ({
     return () => clearTimeout(delayBanner);
   }, [setBannerIsOpen]);
 
+  const showModal = () => {
+    if (!isModalLinkFound) {
+      document.body.classList.add("fides-modal-link-shown");
```

Comment:
Only adds the `fides-modal-link-shown` class to the `body` if the modal link isn't found

Reply:
nit: I think it's slightly safer to do a different classname to not redefine the semantic meaning of this class. In other words, I could see someone already doing this:
```
.fides-modal-link-shown { font-size: 40px }
```

...since today we only ever add that class to the individual link element, that would have been "safe". But if we start adding this class to the `body` that could be disastrous!

Reply:
Ah! I misread the class name, updated to `fides-overlay-modal-link-shown`